tests/console/python310_version_check.pm

```diff
@@ -0,0 +1,54 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: python310
+# Summary: Run python310 testsuite
+# - Test suitable only for SLE15SP4+
+# - Check that Python 3.6 is the main version installed
+# - Check that Python 3.10 is available to install
+# - Run some basic test (man_or_boy.py)
+# Maintainer: QE Core <qe-core@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use version_utils;
+use utils "zypper_call";
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+    if (is_sle('>=15-SP4')) {
+        my $python_ver = script_output("rpm -q python3 | awk -F \'-\' \'{print \$2}\' | awk -F \'.\' \'{print \$1\".\"\$2}\'");
+        if ($python_ver ne "3.6") {
```

Comment:
can you explain a bit the logic behind this? this can change in the future and it will be wrong imo if the test fails in this place

Reply:
Changed to use `package_version_cmp` from `lib/version_utils.pm` library.